### PR TITLE
update ta4j version from 9to11

### PR DESCRIPTION
**Pull request by NikkiKumari**

### CODE REVIEW

**Comment by NikkiKumari** [comment]
Change class name StrategyAnalyser to BackTestAnalyser

File: src/main/java/Strategies/BackTestAnalyser.java [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 
-public class StrategyAnalyser {
+public class BackTestAnalyser {
 
     private AverageProfitableTradesCriterion avgProfitTrades = new AverageProfitableTradesCriterion();
     private AverageProfitCriterion avgProfit = new AverageProfitCriterion();
     private TotalProfitCriterion totalProfit = new TotalProfitCriterion();
     private BuyAndHoldCriterion buyAndHold = new BuyAndHoldCriterion();
     private NumberOfTradesCriterion numTrades = new NumberOfTradesCriterion();
-    private NumberOfTicksCriterion numTicks = new NumberOfTicksCriterion();
```
Comment (NitigyaS) [inline src/main/java/Strategies/BackTestAnalyser.java]: @NikkiKumari Can you also change the following variables name to numBar

File: src/main/java/Strategies/BackTestAnalyser.java [hunk 2]
```diff
@@ -82,9 +82,9 @@ public void printResults(StrategyBuilder sb, boolean showTrades){
 
                     Order exit = record.getTrades().get(i).getExit();
 
-                    Tick entryTick = series.getTick(entry.getIndex());
```
Comment (NitigyaS) [inline src/main/java/Strategies/BackTestAnalyser.java]: Also change these to entryBar and other vars in the project.

File: src/main/java/Strategies/StrategyOne.java [hunk 3]
```diff
@@ -9,21 +9,21 @@
...
-import eu.verdelhan.ta4j.indicators.helpers.MaxPriceIndicator;
-import eu.verdelhan.ta4j.indicators.helpers.MinPriceIndicator;
-import eu.verdelhan.ta4j.trading.rules.CrossedDownIndicatorRule;
-import eu.verdelhan.ta4j.trading.rules.CrossedUpIndicatorRule;
-import eu.verdelhan.ta4j.trading.rules.OverIndicatorRule;
-import eu.verdelhan.ta4j.trading.rules.UnderIndicatorRule;
+import org.ta4j.core.*;
+import org.ta4j.core.indicators.RSIIndicator;
+import org.ta4j.core.indicators.SMAIndicator;
+import org.ta4j.core.indicators.*;
```
Comment (NitigyaS) [inline src/main/java/Strategies/StrategyOne.java:15]: I do not think we need this line.

File: src/main/java/Strategies/StrategyOne.java [hunk 4]
```diff
@@ -49,7 +49,7 @@
     // indicators
     RSIIndicator rsi;
     SMAIndicator sma;
-    SmoothedRSIIndicator smrsi;
+    RSIIndicator smrsi;
```
Comment (NitigyaS) [inline src/main/java/Strategies/StrategyOne.java:52]: not required.

**Review by NitigyaS — CHANGES_REQUESTED** [review]

**Pull request merged**